### PR TITLE
test: Add test for generate_hash and upload_local in storage helpers.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
Reference: #5320

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This adds test for generate_hash function in storage helper.

#### Changes proposed in this pull request:

- Add unit test.

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 @iamareebjamal please review.

**Comment by iamareebjamal** [comment]
Please merge smaller test changes in one PR onlt

**Comment by Anupam-dagar** [comment]
@iamareebjamal I added one more test for the storage helpers.

File: tests/all/integration/api/helpers/test_storage.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    """Contains test for Storage Helpers"""
+
+    def Setup(self):
+        self.app = Setup.create_app()
+
+    """Test local file upload."""
+    @patch('app.api.helpers.storage.upload_local')
+    @patch('app.api.helpers.storage.generate_hash', return_value='hash')
+    @patch('app.api.helpers.storage.get_settings', return_value={'static_domain': 'https://next.eventyay.com'})
+    @patch('app.api.helpers.storage.UploadedFile')
```
Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_storage.py]: If you are using patch, move in unit test

**Comment by Anupam-dagar** [comment]
@iamareebjamal Please review.

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 @iamareebjamal Please review.

File: tests/all/unit/api/helpers/test_storage.py [hunk 2]
```diff
@@ -40,3 +42,44 @@ def test_create_url(self):
...
+    @patch('app.api.helpers.storage.upload_local')
+    @patch('app.api.helpers.storage.generate_hash', return_value='hash')
+    @patch('app.api.helpers.storage.get_settings', return_value={'static_domain': 'https://next.eventyay.com'})
+    @patch('app.api.helpers.storage.UploadedFile')
+    def test_upload_local(self, uploadedfile_object, settings, generated_hash, uploadlocal):
+        expected_response = 'https://next.eventyay.com/media/upload_key/hash/test.pdf'
+        uploadedfile_object.filename = 'test.pdf'
+
+        app = Flask(__name__)
+        with app.app_context():
```
Comment (iamareebjamal) [inline tests/all/unit/api/helpers/test_storage.py]: This does not belong in unit test

Comment (Anupam-dagar) [reply tests/all/unit/api/helpers/test_storage.py]: I need to use app context due to which I wrote this test in integration testing. What is the correct way to proceed if I can not write `app.app_context`

Comment (iamareebjamal) [reply tests/all/unit/api/helpers/test_storage.py]: Move this test to integration

**Comment by Anupam-dagar** [comment]
@iamareebjamal Please review.

File: tests/all/unit/api/helpers/test_storage.py (hunk 2 above)
Comment (iamareebjamal) [reply tests/all/unit/api/helpers/test_storage.py]: Still not moved

Comment (Anupam-dagar) [reply tests/all/unit/api/helpers/test_storage.py]: my bad, I copied the test instead of moving it.

**Comment by Anupam-dagar** [comment]
@iamareebjamal I guess it's good to go now?

**Comment by Anupam-dagar** [comment]
@iamareebjamal Please merge.

**Pull request merged**